### ISSUE TITLE
is this actually be: "path_match":"span.attributes.*"

### ISSUE THREAD

**Issue by rayliutoronto** [opendistro-for-elasticsearch/data-prepper#613]
https://github.com/opendistro-for-elasticsearch/data-prepper/blob/578cadf952b99e5907765faf7d4d3dc78ce12ba4/data-prepper-plugins/elasticsearch/src/main/resources/otel-v1-apm-span-index-template.json#L19

**Comment by chenqi0805** [comment]
@rayliutoronto Thanks for your interest in looking into data-prepper. Yes, it is a bug in our index template by double-checking the resulting index mapping:

```
GET otel-v1-apm-span-000001/_mapping
...
"resource" : {
          "properties" : {
            "attributes" : {
              "properties" : {
                "host@hostname" : {
                  "type" : "keyword"
                },
                "os@description" : {
                  "type" : "keyword"
                },
                "os@type" : {
                  "type" : "keyword"
                },
                "process@command_line" : {
                  "type" : "keyword"
                },
                "process@executable@path" : {
                  "type" : "keyword"
                },
                "process@pid" : {
                  "type" : "keyword"
                },
                "process@runtime@description" : {
                  "type" : "keyword"
                },
                "process@runtime@name" : {
                  "type" : "keyword"
                },
                "process@runtime@version" : {
                  "type" : "keyword"
                },
                "service@instance@id" : {
                  "type" : "keyword"
                },
                "service@name" : {
                  "type" : "keyword"
                },
                "telemetry@auto@version" : {
                  "type" : "keyword"
                },
                "telemetry@sdk@language" : {
                  "type" : "keyword"
                },
                "telemetry@sdk@name" : {
                  "type" : "keyword"
                },
                "telemetry@sdk@version" : {
                  "type" : "keyword"
                }
              }
            }
          }
        },
        "serviceName" : {
          "type" : "keyword"
        },
        "span" : {
          "properties" : {
            "attributes" : {
              "properties" : {
                "db@name" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "db@statement" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "db@system" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "db@user" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "http@client_ip" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "http@flavor" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "http@host" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "http@method" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "http@route" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "http@scheme" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "http@server_name" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "http@status_code" : {
                  "type" : "long"
                },
                "http@status_text" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "http@target" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "http@url" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "http@user_agent" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "net@host@port" : {
                  "type" : "long"
                },
                "net@peer@ip" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "net@peer@name" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                },
                "net@peer@port" : {
                  "type" : "long"
                },
                "thread@id" : {
                  "type" : "long"
                },
                "thread@name" : {
                  "type" : "text",
                  "fields" : {
                    "keyword" : {
                      "type" : "keyword",
                      "ignore_above" : 256
                    }
                  }
                }
              }
            }
          }
        },
```

**Comment by chenqi0805** [comment]
Closed via #624
